packages/lib/src/types/data.ts

```diff
@@ -1,5 +1,11 @@
 import { BlocksDefs } from "../ui/blocks/blocks"
 
+export interface Image {
```

Comment:
TOhle není úplně šťastné. Spíš pro to založ nový soubor `Image.ts` a tento přejmenuj na `Page.ts`.